cmd/deploy/deploy.go

```diff
@@ -199,6 +212,18 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		}
 	}
 
+	if opts.Wait {
+		spinner := utils.NewSpinner("Waiting to be fully deployed...")
+		spinner.Start()
+		defer spinner.Stop()
+		if err := dc.waitUntilDeployed(ctx, opts); err != nil {
```

Comment:
Could you attach a video?
It would be nice if for each resource, we show an spinner in a multi-line. Something like:
```
* Deployment 'vote' is progressing...
* Statefulset redis is progressing...
```
that becomes a tick when any of them get's deployed:
```
 ✓  Deployment 'vote' is running
* Statefulset redis is progressing...
```

Reply:
Right now it's like this. I didn't use our color palette because the output continue spinning, but we could add it.
https://user-images.githubusercontent.com/25170843/144068556-3dae1a32-2957-447f-9cde-412d0a9cdba9.mov